From b9003104b483414ecf5a911ab15a4eaac394ae3d Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 4 Jul 2019 22:12:09 -0700
Subject: [PATCH 0640/1851] v4l: xilinx: sdirxss: Make functions static inline

commit f349b72b06da73698b56076583fcfa1c86fbc00f from
https://github.com/Xilinx/linux-xlnx.git

Make some of the functions static inline as they are one line functions.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-sdirxss.c | 18 ++++++++++--------
 1 file changed, 10 insertions(+), 8 deletions(-)

diff --git a/drivers/media/platform/xilinx/xilinx-sdirxss.c b/drivers/media/platform/xilinx/xilinx-sdirxss.c
index 9eeda66c363a..963d446c226f 100644
--- a/drivers/media/platform/xilinx/xilinx-sdirxss.c
+++ b/drivers/media/platform/xilinx/xilinx-sdirxss.c
@@ -491,12 +491,12 @@ static inline void xsdirxss_set(struct xsdirxss_core *xsdirxss, u32 addr,
 	xsdirxss_write(xsdirxss, addr, xsdirxss_read(xsdirxss, addr) | set);
 }
 
-static void xsdirx_core_disable(struct xsdirxss_core *core)
+static inline void xsdirx_core_disable(struct xsdirxss_core *core)
 {
 	xsdirxss_clr(core, XSDIRX_RST_CTRL_REG, XSDIRX_RST_CTRL_SS_EN_MASK);
 }
 
-static void xsdirx_core_enable(struct xsdirxss_core *core)
+static inline void xsdirx_core_enable(struct xsdirxss_core *core)
 {
 	xsdirxss_set(core, XSDIRX_RST_CTRL_REG, XSDIRX_RST_CTRL_SS_EN_MASK);
 }
@@ -604,7 +604,7 @@ static void xsdirx_setedherrcnttrigger(struct xsdirxss_core *core, u32 enable)
 	xsdirxss_write(core, XSDIRX_EDH_ERRCNT_EN_REG, val);
 }
 
-static void xsdirx_setvidlockwindow(struct xsdirxss_core *core, u32 val)
+static inline void xsdirx_setvidlockwindow(struct xsdirxss_core *core, u32 val)
 {
 	/*
 	 * The video lock window is the amount of time for which the
@@ -614,12 +614,12 @@ static void xsdirx_setvidlockwindow(struct xsdirxss_core *core, u32 val)
 	xsdirxss_write(core, XSDIRX_VID_LOCK_WINDOW_REG, val);
 }
 
-static void xsdirx_disableintr(struct xsdirxss_core *core, u32 mask)
+static inline void xsdirx_disableintr(struct xsdirxss_core *core, u32 mask)
 {
 	xsdirxss_clr(core, XSDIRX_IER_REG, mask);
 }
 
-static void xsdirx_enableintr(struct xsdirxss_core *core, u32 mask)
+static inline void xsdirx_enableintr(struct xsdirxss_core *core, u32 mask)
 {
 	xsdirxss_set(core, XSDIRX_IER_REG, mask);
 }
@@ -634,12 +634,13 @@ static void xsdirx_globalintr(struct xsdirxss_core *core, bool flag)
 			     XSDIRX_GLBL_INTR_EN_MASK);
 }
 
-static void xsdirx_clearintr(struct xsdirxss_core *core, u32 mask)
+static inline void xsdirx_clearintr(struct xsdirxss_core *core, u32 mask)
 {
 	xsdirxss_set(core, XSDIRX_ISR_REG, mask);
 }
 
-static void xsdirx_vid_bridge_control(struct xsdirxss_core *core, bool enable)
+static void xsdirx_vid_bridge_control(struct xsdirxss_core *core,
+				      bool enable)
 {
 	if (enable)
 		xsdirxss_set(core, XSDIRX_RST_CTRL_REG,
@@ -649,7 +650,8 @@ static void xsdirx_vid_bridge_control(struct xsdirxss_core *core, bool enable)
 			     XSDIRX_RST_CTRL_SDIRX_BRIDGE_ENB_MASK);
 }
 
-static void xsdirx_axis4_bridge_control(struct xsdirxss_core *core, bool enable)
+static void xsdirx_axis4_bridge_control(struct xsdirxss_core *core,
+					bool enable)
 {
 	if (enable)
 		xsdirxss_set(core, XSDIRX_RST_CTRL_REG,
-- 
2.31.1

